lib/rails/generators/intercom/config/intercom.rb.erb

```diff
@@ -71,6 +71,7 @@ IntercomRails.config do |config|
   # == User -> Company association
   # A Proc that given a user returns an array of companies
   # that the user belongs to.
+  # This is uniquely used for importing users using rake command but it will not inject a list of company in the intercom widget
```

Comment:
I'd go with `only used` rather than `uniquely used` and `list of companies` (plural)